src/AutoRest.CSharp/MgmtTest/Generation/MgmtExtensionTestWriter.cs

```diff
@@ -0,0 +1,140 @@
+        }
+
+        protected void WriteTesterCtors()
+        {
+            _writer.Line();
+            using (_writer.Scope($"public {TypeNameOfThis}(bool isAsync): base(isAsync, RecordedTestMode.Record)"))
+            {
+                _writer.Line($"ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;");
+                _writer.Line($"System.Environment.SetEnvironmentVariable(\"RESOURCE_MANAGER_URL\", $\"https://localhost:8443\");");
+            }
+        }
+
+        protected void WriteMethodTest(MgmtClientOperation clientOperation, bool async, bool isLroOperation)
+        {
+            Debug.Assert(clientOperation != null);
+            var methodName = clientOperation.Name;
+            int exampleIdx = 0;
+            foreach ((var branch, var operation) in GetSortedOperationMappings(clientOperation))
+            {
+                var exampleGroup = MgmtBaseTestWriter.FindExampleGroup(operation);
+                if (exampleGroup is null || exampleGroup.Examples.Count() == 0)
+                    return;
+
+                foreach (var exampleModel in exampleGroup?.Examples ?? Enumerable.Empty<ExampleModel>())
+                {
+                    WriteTestDecorator();
+                    var testCaseSuffix = exampleIdx > 0 ? (exampleIdx + 1).ToString() : String.Empty;
+                    _writer.Append($"public {GetAsyncKeyword(async)} {MgmtBaseTestWriter.GetTaskOrVoid(async)} {methodName}{testCaseSuffix}()");
+                    using (_writer.Scope())
+                    {
+                        _writer.LineRaw($"// Example: {exampleModel.Name}");
+                        WriteOperationInvocation(clientOperation, operation, exampleModel, async, isLroOperation);
+                    }
+                    _writer.Line();
+                    exampleIdx++;
+                }
+            }
+        }
+
+        public void WriteOperationInvocation(MgmtClientOperation clientOperation, MgmtRestOperation restOperation, ExampleModel exampleModel, bool async, bool isLroOperation)
+        {
+            var testMethodName = CreateMethodName(clientOperation.Name, async);
+            var resourceIdentifierParams = ComposeResourceIdentifierParams(restOperation.RequestPath, exampleModel);
+            var subscriptionVariableName = new CodeWriterDeclaration(extensions.Type.Name.FirstCharToLowerCase());
+            var subscriptionRequestPath = GetSubscriptionRequestPath(restOperation.RequestPath.SerializedPath, exampleModel);
+            _writer.Line($@"var {subscriptionVariableName:D} = GetArmClient().GetSubscription(new {typeof(Azure.Core.ResourceIdentifier)}({FormatResourceId(subscriptionRequestPath).RefScenariDefinedVariables(scenarioVariables)}));");
+            List<KeyValuePair<string, FormattableString>> parameterValues = WriteOperationParameters(clientOperation.MethodParameters.ToArray().AsSpan().Slice(1).ToArray(), exampleModel);
```

Comment:
What is this Linq invocation doing? If this is want to take the parameters starting from index `1`, you could do this which is more simple and comprehensive:
```suggestion
            List<KeyValuePair<string, FormattableString>> parameterValues = WriteOperationParameters(clientOperation.MethodParameters.Skip(1), exampleModel);
```